hindi/India/India.md

```diff
@@ -0,0 +1,16 @@
+# <h1 align="center">NAMASTE!!</h1>
+------------------------------------------------- 
+## <center>Welcome to India</center>
+
+>India is one of the oldest civilizations in the world with a kaleidoscopic variety and rich cultural heritage. It has achieved all-round socio-economic progress during the last 67 years of its Independence. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.